server/testsupport/inductionResponseTestDataBuilder.ts

```diff
@@ -0,0 +1,33 @@
+import type { InductionResponse } from 'educationAndWorkPlanApiClient'
+
+const aValidInductionResponse = (options?: {
+  createdBy?: string
+  createdByDisplayName?: string
+  createdAt?: string
+  createdAtPrison?: string
+  updatedBy?: string
+  updatedByDisplayName?: string
+  updatedAt?: string
+  updatedAtPrison?: string
+}): InductionResponse => {
+  return {
+    reference: '814ade0a-a3b2-46a3-862f-79211ba13f7b',
+    createdBy: options?.createdByDisplayName || 'asmith_gen',
+    createdByDisplayName: options?.createdByDisplayName || 'Alex Smith',
+    createdAt: options?.createdAt || '2023-06-19T09:39:44Z',
+    createdAtPrison: options?.createdAtPrison || 'MDI',
+    updatedBy: options?.updatedBy || 'asmith_gen',
+    updatedByDisplayName: options?.updatedByDisplayName || 'Alex Smith',
+    updatedAt: options?.updatedAt || '2023-06-19T09:39:44Z',
+    updatedAtPrison: options?.updatedAtPrison || 'MDI',
+    workOnRelease: undefined,
```

Comment:
These child fields will be built upon and improved over the course of subsequent PRs for this story. For the purpose of this PR and the unit test for the client method, this test data builder is sufficient as it is for now.

Reply:
I recommend something like `aValidInductionResponseForAPrisonLookingForWork()` etc in future.

Reply:
That would actually solve the dilemma I was having 👍 (I was struggling with how to write an options object to allow the caller to specify how it wanted the object created for it's specific test)
Good shout, I'll rename it in my next PR and build upon it 👍 
Thanks